app/schedulers/delete_unverified_users_cron_job.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+import config
+
+
+def delete_unverified_users_job():
+    """
+       This function iterates of all the users and
+       checks if the email is verified. If email is not verified
+       then we are checking whether the specified threshold has passed
+       since registration. If yes, user is deleted from the database.
+       """
+
+    from run import application
+
+    with application.app_context():
+        from app.database.models.user import UserModel
+        all_users = UserModel.query.all()
```

Comment:
filter by unverified, please

Reply:
Hi, umm...how can I do that? Should I make a raw SQL query? I already filter users to retrieve only unverified ones a few lines below.

Reply:
use something like lambda filter